package.json

```diff
@@ -48,6 +48,7 @@
     "ember-cli-shims": "^1.1.0",
     "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "^2.0.0",
+    "ember-cli-version-checker": "^2.1.0",
```

Comment:
i'm curious if this is going to work or not in consuming apps with this not being in `dependencies`

Reply:
Just tested it on 2 apps.
Ember 2.8:
```
import resolver from './helpers/resolver';
import './helpers/flash-message';


import {
  setResolver
} from 'ember-qunit';

setResolver(resolver);
```
Ember 3:
```
import Application from '../app';
import config from '../config/environment';
import { setApplication } from '@ember/test-helpers';
import { start } from 'ember-qunit';
import './helpers/flash-message';


setApplication(Application.create(config.APP));

start();
```

Seems to be working

Reply:
I could definitely be wrong, but I wonder if this addon already exists in most scenarios given how common its usage is throughout the ember ecosystem.

Reply:
For instance yarn.lock didn’t need to be updated for it to be added here. We can always fix is someone notices anything not working.

Reply:
haha, yeah: https://github.com/emberjs/ember.js/blob/master/package.json#L55

It's a dependency of `ember` so, I think we're getting it for free :P

Reply:
TIL!